### ISSUE TITLE
sccache should work on Windows

### ISSUE THREAD

**Issue by brenca** [electron/electron#15090]
* Output of `node_modules/.bin/electron --version`: 4.0.0
* Operating System (Platform and Version): Windows 10
* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable):

**Expected Behavior**
`sccache` should work on Windows

**Actual behavior**
`sccache` produces compilation errors

**To Reproduce**
Try to build `electron` on Windows with `sccache` enabled

**Additional Information**
Currently we are waiting on two `sccache` issues to be resolved, after that `sccache` should work on every platform.

The issues are:
https://github.com/mozilla/sccache/issues/246
https://github.com/mozilla/sccache/issues/264

**Comment by MarshallOfSound** [comment]
@brenca I did actually get sccache working on windows during the C68 upgrade but had other issues with C68 so left it alone for a while.

The first issue you linked is fixed in C68 and above and the second one we can patch around, Google actually doesn't use that flag when goma is active so we should be able to turn it off for sccache builds as well.

I can try ressurect by WIP on my windows machine next week

**Comment by brenca** [comment]
@MarshallOfSound That would be awesome, would love to use `sccache` locally to speed things up.

**Comment by MarshallOfSound** [comment]
Refs: https://cs.chromium.org/chromium/src/build/config/win/BUILD.gn?l=118

**Comment by fxb** [comment]
I'd clarify the title of this issue to read "sccache should work with clang-cl on Windows", otherwise it is misleading, since it does work with MSVC on Windows.

**Comment by nornagon** [comment]
Electron isn't built with MSVC so it's kind of moot that sccache works with MSVC :)

**Comment by fxb** [comment]
@nornagon I certainly know that, but it doesn't change that the issue title is misleading. :)

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by jviotti** [comment]
@MarshallOfSound 

>  I did actually get sccache working on windows during the C68 upgrade but had other issues with C68 so left it alone for a while.

Do you roughly remember what you did? We're operating an Electron fork with sccache and the lack of Windows support is a bit a problem. We did try disabling `/Brepro` as suggested in one of the issues and various other things, but we couldn't get more than a few hits so far.

> I can try resurrect my WIP on my windows machine next week

It would be awesome if you can point me to whatever you have, and we can try to revive it. I understand that sccache support is no longer relevant due to Goma, but we are interested in internally maintaining support for it any way!

**Comment by jviotti** [comment]
For the record, we got sccache to work fine on Windows on our Electron v11 fork. We will be sharing some details and potential contribution soon if there is interest on maintaining sccache support on a community-effort

**Comment by jviotti** [comment]
This is what works for us on CircleCI and local environments for Electron.js v11: https://github.com/electron/electron/pull/29896

**Comment by jviotti** [comment]
The CL landed here: https://chromium.googlesource.com/chromium/src/+/9c7622d23e56390856cc8ce287b4537c77147dad
